connector/src/main/resources/application-sandbox.properties

```diff
@@ -1,6 +1,7 @@
 spring.profiles=sandbox
 server.port=8080
 wsPort=8088
+tcpPort=9088
```

Comment:
Maybe not directly linked to this pull request, but we should define a naming strategy for these configs. Like for example, `linker.messaging.tcp.port`

Reply:
we need to define the naming convention , we leave it as it is at the moment